scripts/build-cloud-hypervisor.sh

```diff
@@ -1,203 +1,80 @@
 #!/bin/bash
 
-# Copyright(c) 2011-2024 The Maintainers of Nanvix.
+# Copyright(c) The Maintainers of Nanvix.
 # Licensed under the MIT License.
 
-#===================================================================================================
-# Script Arguments
-#===================================================================================================
+# This script builds our fork of cloud-hypervisor as well as the linuxd kernel used for direct boot.
 
-RULE=${1:-build}
-PREFIX=${2:-$PWD/toolchain}
+set -euo pipefail
 
 #===================================================================================================
 # Global Variables
 #===================================================================================================
 
 NANVIX_HOME=`git rev-parse --show-toplevel`
-CONTRIB_DIR=${PREFIX}/src
-CLOUD_HYPERVISOR_HOME=${CONTRIB_DIR}/cloud-hypervisor
-CLOUD_HYPERVISOR_REPOSITORY=https://github.com/nanvix/cloud-hypervisor
-CLOUD_HYPERVISOR_COMMIT=3d88996e5b509b0840b8912142104ddf86a10eeb
-
-BINARIES_DIR=${NANVIX_HOME}/bin
-SCRIPTS_DIR=${NANVIX_HOME}/scripts
-SERVICE_FILE=${SCRIPTS_DIR}/linuxd.service
-SERVICE_ELF_BASENAME=linuxd.elf
-SERVICE_ELF=${BINARIES_DIR}/${SERVICE_ELF_BASENAME}
-IMAGES_DIR=${NANVIX_HOME}/images
-# The VM will be configure to use the IP 192.168.249.2 when using this specific MAC Address
-GUEST_MAC_ADDRESS=12:34:56:78:90:ab
-# This is the IP that the VM will use when using this specific MAC Address
-GUEST_TAP_IP_ADDRESS=192.168.249.2
-# This is the IP that host applications can bind to so that the guest applications can connect to them
-HOST_TAP_IP_ADDRESS=192.168.249.3
-IMAGE_NAME=custom-ubuntu.raw
-
-#===================================================================================================
-# Distclean
-#===================================================================================================
+CONTRIB_DIR="${NANVIX_HOME}/contrib"
+IMAGES_DIR="${NANVIX_HOME}/images"
+L2_SYSTEM_VM_KERNEL="${IMAGES_DIR}/l2_sysvm_vmlinux.bin"
 
-distclean() {
-	cd ${CLOUD_HYPERVISOR_HOME}
-	git clean -fdx
-}
+# Cloud-hypervisor variables.
+CLOUD_HYPERVISOR_HOME="${CONTRIB_DIR}/cloud-hypervisor"
+CLOUD_HYPERVISOR_REPOSITORY="https://github.com/nanvix/cloud-hypervisor"
+# TODO: update me!
```

Comment:
Must update this line once we merge https://github.com/nanvix/cloud-hypervisor/pull/1